telethon/client/telegrambaseclient.py

```diff
@@ -551,7 +551,7 @@ def _auth_key_callback(self, auth_key):
             self._exported_sessions[cdn_redirect.dc_id] = session
 
         self._log[__name__].info('Creating new CDN client')
-        client = TelegramBareClient(
+        client = TelegramBaseClient(
```

Comment:
I know you were getting an error for this but just above there's a `raise NotImplementedError`, so it was fine to leave it ;)

Reply:
Yep, I noticed it. I just got warning from pylint. Should I revert this change?

Reply:
I prefer if you do revert it since it's not relevant to this PR yes.